src/node.cc

```diff
@@ -2409,18 +2406,12 @@ void FatalException(Isolate* isolate,
     if (fatal_try_catch.HasCaught()) {
       // the fatal exception function threw, so we must exit
       ReportException(env, fatal_try_catch);
-      exit_code = 7;
-    }
-
-    if (exit_code == 0 && false == caught->BooleanValue()) {
+      exit(7);
+    } else if (false == caught->BooleanValue()) {
```

Comment:
Can we either un-yoda-ize this and make it use the non-deprecated overload of `BooleanValue`, or switch to `if (caught->IsFalse())`?

Reply:
Addressed